### DISCUSSION TITLE
Load HTML from API containing <v-chip>hello</v-chip> and render as Chip component

### DISCUSSION THREAD

**Discussion by ligne13** [vuetifyjs/vuetify#14206] [General] [answered]
Hi,

I load HTML content from my API. This content contains the text : `<v-chip>Hello</v-chip>`. How can I render this to a Vuetify Chip component when displaying the content in my Vue app ?

Thank you.

**Comment by jacekkarczmarczyk** [comment, marked as answer]
https://codepen.io/jkarczm/pen/mdwgbMp?editors=1010
Not the best practice though

**Comment by ligne13** [comment]
Thank you !
I know this is not ideal but for now this will be fine :)
